src/models/edge.ts

```diff
@@ -256,6 +274,25 @@ abstract class Edge<N extends INodeBase, E extends IEdgeBase> implements IEdge<N
 
     return color;
   }
+
+  getLineDashPattern(): number[] | null {
+    const lineStyle: IEdgeLineStyle | undefined = this.style.lineStyle;
+
+    if (lineStyle === undefined || lineStyle.type === EdgeLineStyleType.SOLID) {
+      return null;
+    }
+
+    switch (lineStyle.type) {
+      case EdgeLineStyleType.DASHED:
+        return DEFAULT_DASHED_LINE_PATTERN;
+      case EdgeLineStyleType.DOTTED:
+        return DEFAULT_DOTTED_LINE_PATTERN;
+      case EdgeLineStyleType.CUSTOM:
+        return isArrayOfNumbers(lineStyle.pattern) ? lineStyle.pattern : null;
```

Comment:
Nothing to change here, but just a general comment that once node and edge style have a setter for styles `setStyle` (what 1.0.0 will require), this check will be done there because the check is done once then.